lib/logflare_web/live/query_live.ex

```diff
@@ -142,13 +143,18 @@ defmodule LogflareWeb.QueryLive do
   def mount(%{}, %{"user_id" => user_id}, socket) do
     user = Users.get(user_id)
 
+    endpoints = Endpoints.list_endpoints_by(user_id: user.id)
+    alerts = Alerting.list_alert_queries_by_user_id(user.id)
```

Comment:
Currently these variables are not used, I expect they should be put in assigns below.